### PR TITLE
Adding empty views

### PR DESCRIPTION
**Pull request by coreyfloyd**
Empty view support for feed, articles, and search.

View is shown using a category on UIViewController. 

Using an un-speced heuristic for the feed… showing empty view if half of the requests fail. Not really sure about that.

### CODE REVIEW

**Comment by montehurd** [comment]
Small font-size diff:

screen shot 2015-12-10 at 2 59 34 pm

**Comment by montehurd** [comment]
I'm occasionally seeing flicker of empty view just after tapping links in articles.

File: Wikipedia/Localizations/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -362,4 +362,10 @@
 "welcome-usage-reports-button-title" = "Done";
 "welcome-usage-reports-privacy-policy-button-text" = "Your privacy is important to us";
 
+"empty-no-feed-title" = "No Internet Connection";
+"empty-no-feed-message" = "You can see your recommended articles when you have internet";
```
Comment (montehurd) [inline Wikipedia/Localizations/en.lproj/Localizable.strings]: Potential tweak:
"Internet connection required for article recommendations"

File: Wikipedia/Localizations/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -362,4 +362,10 @@
...
+"empty-no-feed-action-message" = "You can read saved pages meanwhile";
```
Comment (montehurd) [inline Wikipedia/Localizations/en.lproj/Localizable.strings]: Or maybe...
"Saved pages can be read without an Internet connection."

**Comment by coreyfloyd** [comment]
@montehurd updated font size. 

As far as copy - thats Nirzar and Josh… I'm just inserting it.

**Comment by montehurd** [comment]
@coreyfloyd cool, your call.

**Comment by montehurd** [comment]
@coreyfloyd also getting quick flicker of could not load message on peek:

screen shot 2015-12-10 at 3 13 44 pm

**Comment by montehurd** [comment]
Ah just saw your commit... retesting...

**Comment by montehurd** [comment]
Hmm still seeing the flicker on peek and link tap on device... retesting on sim...

**Comment by coreyfloyd** [comment]
hmm… ok - ill have to investigate more later

**Comment by montehurd** [comment]
On simulator it's not very apparent with WIFI, but when I turned on network link conditioner for a slow network it happens regularly.

**Pull request merged**